packages/compile-common/src/sources.ts

```diff
@@ -18,6 +18,10 @@ export function collectSources(
   replacement: string = "/"
 ): CollectedSources {
   const mappedResults = Object.entries(originalSources)
+    .filter(([originalSourcePath, _]) =>
```

Comment:
Is `_` necessary here?

Reply:
Technically not, I suppose, but I feel like it would be confusing without it.  We've got a pair `[key, value]` and we're filtering on just the key, so I put named the value `_`; technically one could omit the value and just write `[key]`, but that seems more confusing to me; it makes it look like only the key is present.

Reply:
sounds good!